### PR TITLE
24 ratings db

### LINKED ISSUE #24: Create db for ratings
Create Ratings table and seed some data

### PR DESCRIPTION
**Pull request by danielle-baek**
-seed and migration for ratings table is done

closes #24

### CODE REVIEW

File: migrations/20181206104452_ratings.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
+exports.up = knex =>
+  knex.schema.createTable('ratings', t => {
+    t.increments('id').primary()
+    t.integer('log_id').unique()
```
Comment (chrislaupama) [inline migrations/20181206104452_ratings.js]: I think this is supposed to be...

.references(logs.id')

File: migrations/20181206104452_ratings.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+    t.integer('activity_id').unique()
```
Comment (chrislaupama) [inline migrations/20181206104452_ratings.js]: I think this is supposed to be...

.references(activities.id')

File: migrations/20181206104452_ratings.js (hunk 1 above)
Comment (chrislaupama) [reply migrations/20181206104452_ratings.js]: So like this:
t.integer('log_id').references(logs.id')

File: migrations/20181206104452_ratings.js (hunk 2 above)
Comment (chrislaupama) [reply migrations/20181206104452_ratings.js]: So like this:
t.integer('activity_id').references(activities.id')

File: migrations/20181206104443_logs.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
+exports.up = knex =>
+  knex.schema.createTable('logs', t => {
+    t.increments('id').primary()
+    t.integer('user_id').unique()
```
Comment (chrislaupama) [inline migrations/20181206104443_logs.js]: Need to change this to:
t.integer('user_id').references(users.id')

**Review by chrislaupama — CHANGES_REQUESTED** [review]
Some changes to be made, but good work.

Remember, if a DB file is referencing another table, it should have the .references(table.id) at the end, not unique.

File: migrations/20181206104443_logs.js (hunk 3 above)
Comment (danielle-baek) [reply migrations/20181206104443_logs.js]: Thanks Chris :) I fixed all FKs

**Review by chrislaupama — APPROVED** [review]
Awesome work!

**Pull request merged**